Makefile

```diff
@@ -74,6 +74,10 @@ container-build-operator: csv-generator
 container-build-registry:
 	docker build -f build/Dockerfile.registry -t $(IMAGE_REGISTRY)/$(REGISTRY_IMAGE):$(IMAGE_TAG) .
 
+test-courier:
+	echo "bundle dir: $(BUNDLE_DIR_VERSION)"
+	export BUNDLE_DIR_VERSION; docker build -f build/Dockerfile.test-courier -t test-courier . --build-arg dir=$(BUNDLE_DIR_VERSION)
```

Comment:
I'm unclear how basing courier checks on a docker image is related to the goal of making them mandatory.  `operator-courier` seems to be used similarly to `operator-sdk` which isn't containerized, wouldn't adding `operator-courier verify --ui_validate_io $dir` to the Makefile achieve the same thing?

Can you explain your thinking please?

Reply:
the  script we had before this change was checking if the courier check executable is installed, if it is not installed then the checks were skipped, so the checks were conditioned on the presence of the courier check executable on the current system. I added a docker that sets up the courier check executable (actually that's what they do in the hco build scripts).

Reply:
That doesn't answer my question.

Why are `operator-sdk` (installed locally and invoked directly from the Makefile) and `operator-courier` (dockerized) treated differently?

Reply:
Sorry for the misunderstanding. the courier tool is installed via pip and I couldn't get pip --user to  work in this case. (pip --user installs the package for the current user). I am about to put in a commit that adds a comment to explain the circumstances.